Index: qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/qmlmin/tst_qmlmin.cpp
===================================================================
--- qtdeclarative-opensource-src-5.6.0-rc.orig/tests/auto/qml/qmlmin/tst_qmlmin.cpp
+++ qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/qmlmin/tst_qmlmin.cpp
@@ -67,7 +67,7 @@ tst_qmlmin::tst_qmlmin()
 
 void tst_qmlmin::initTestCase()
 {
-    qmlminPath = QLibraryInfo::location(QLibraryInfo::BinariesPath) + QLatin1String("/qmlmin");
+    qmlminPath = QLatin1String("../../../../bin") + QLatin1String("/qmlmin");
 #ifdef Q_OS_WIN
     qmlminPath += QLatin1String(".exe");
 #endif
Index: qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/qmlplugindump/tst_qmlplugindump.cpp
===================================================================
--- qtdeclarative-opensource-src-5.6.0-rc.orig/tests/auto/qml/qmlplugindump/tst_qmlplugindump.cpp
+++ qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/qmlplugindump/tst_qmlplugindump.cpp
@@ -59,7 +59,7 @@ tst_qmlplugindump::tst_qmlplugindump()
 
 void tst_qmlplugindump::initTestCase()
 {
-    qmlplugindumpPath = QLibraryInfo::location(QLibraryInfo::BinariesPath);
+    qmlplugindumpPath = QLatin1String("../../../../bin");
 
 #if defined(Q_OS_WIN)
     qmlplugindumpPath += QLatin1String("/qmlplugindump.exe");
Index: qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmldebugjs/tst_qqmldebugjs.cpp
===================================================================
--- qtdeclarative-opensource-src-5.6.0-rc.orig/tests/auto/qml/debugger/qqmldebugjs/tst_qqmldebugjs.cpp
+++ qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmldebugjs/tst_qqmldebugjs.cpp
@@ -829,7 +829,7 @@ void tst_QQmlDebugJS::cleanupTestCase()
 void tst_QQmlDebugJS::init(const QString &qmlFile, bool blockMode, bool restrictServices)
 {
     connection = new QQmlDebugConnection();
-    process = new QQmlDebugProcess(QLibraryInfo::location(QLibraryInfo::BinariesPath) + "/qmlscene", this);
+    process = new QQmlDebugProcess(QLatin1String("../../../../../bin") + "/qmlscene", this);
     client = new QJSDebugClient(connection);
 
     const char *args = 0;
Index: qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmlenginedebuginspectorintegrationtest/tst_qqmlenginedebuginspectorintegration.cpp
===================================================================
--- qtdeclarative-opensource-src-5.6.0-rc.orig/tests/auto/qml/debugger/qqmlenginedebuginspectorintegrationtest/tst_qqmlenginedebuginspectorintegration.cpp
+++ qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmlenginedebuginspectorintegrationtest/tst_qqmlenginedebuginspectorintegration.cpp
@@ -102,7 +102,7 @@ void tst_QQmlEngineDebugInspectorIntegra
                                     QString());
 
     // ### Still using qmlscene because of QTBUG-33376
-    m_process = new QQmlDebugProcess(QLibraryInfo::location(QLibraryInfo::BinariesPath)
+    m_process = new QQmlDebugProcess(QLatin1String("../../../../../bin")
                                      + "/qmlscene", this);
     m_process->start(QStringList() << argument << testFile("qtquick2.qml"));
     QVERIFY2(m_process->waitForSessionStart(),
Index: qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmlinspector/tst_qqmlinspector.cpp
===================================================================
--- qtdeclarative-opensource-src-5.6.0-rc.orig/tests/auto/qml/debugger/qqmlinspector/tst_qqmlinspector.cpp
+++ qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmlinspector/tst_qqmlinspector.cpp
@@ -84,7 +84,7 @@ void tst_QQmlInspector::startQmlscenePro
             .arg(restrictServices ? QStringLiteral(",services:QmlInspector") : QString());
 
     // ### This should be using qml instead of qmlscene, but can't because of QTBUG-33376 (same as the XFAIL testcase)
-    m_process = new QQmlDebugProcess(QLibraryInfo::location(QLibraryInfo::BinariesPath) + "/qmlscene", this);
+    m_process = new QQmlDebugProcess(QLatin1String("../../../../../bin") + "/qmlscene", this);
     m_process->start(QStringList() << argument << testFile("qtquick2.qml"));
     QVERIFY2(m_process->waitForSessionStart(),
              "Could not launch application, or did not get 'Waiting for connection'.");
Index: qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmlprofilerservice/tst_qqmlprofilerservice.cpp
===================================================================
--- qtdeclarative-opensource-src-5.6.0-rc.orig/tests/auto/qml/debugger/qqmlprofilerservice/tst_qqmlprofilerservice.cpp
+++ qtdeclarative-opensource-src-5.6.0-rc/tests/auto/qml/debugger/qqmlprofilerservice/tst_qqmlprofilerservice.cpp
@@ -359,7 +359,7 @@ void QQmlProfilerClient::messageReceived
 void tst_QQmlProfilerService::connect(bool block, const QString &testFile, bool restrictServices)
 {
     // ### Still using qmlscene due to QTBUG-33377
-    const QString executable = QLibraryInfo::location(QLibraryInfo::BinariesPath) + "/qmlscene";
+    const QString executable = QLatin1String("../../../../../bin") + "/qmlscene";
     QStringList arguments;
     arguments << QString::fromLatin1("-qmljsdebugger=port:%1,%2%3%4")
                  .arg(STR_PORT_FROM).arg(STR_PORT_TO)
